swift-sdk/Internal/api-client/Request/RequestCreator.swift

```diff
@@ -424,6 +424,10 @@ struct RequestCreator {
     // MARK: - Embedded Messaging Request Calls
     
     func createGetEmbeddedMessagesRequest() -> Result<IterableRequest, IterableError> {
+        return createGetEmbeddedMessagesRequest(placementIds: [])
+    }
+    
+    func createGetEmbeddedMessagesRequest(placementIds: [Int]) -> Result<IterableRequest, IterableError> {
```

Comment:
```suggestion
    func createGetEmbeddedMessagesRequest(placementIds: [Int]? = []) -> Result<IterableRequest, IterableError> {
```


I would just update the method here to take in a default value of an empty array instead of cascading the logic.


The only other place where `createGetEmbeddedMessagesRequest()` is called is at 

`let result = createRequestCreator().flatMap { $0.createGetEmbeddedMessagesRequest(placementIds: placementIds ?? []) }` 

which can also be simplified to 

`let result = createRequestCreator().flatMap { $0.createGetEmbeddedMessagesRequest(placementIds: placementIds) }` 

because `createGetEmbeddedMessagesRequest` gets an optional array now